Philips.CodeAnalysis.MaintainabilityAnalyzers/NoSpaceInFilenameAnalyzer.cs

```diff
@@ -0,0 +1,69 @@
+﻿// © 2020 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using System.Collections.Immutable;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+
+using Philips.CodeAnalysis.Common;
+
+namespace Philips.CodeAnalysis.MaintainabilityAnalyzers
+{
+	/// <summary>
+	/// Report on file names with spaces, as these tend to cause scripting issues.
+	/// </summary>
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class NoSpaceInFilenameAnalyzer : DiagnosticAnalyzer
+	{
+		/// <summary>
+		/// Diagnostic for file names with spaces.
+		/// </summary>
+		private const string Title = "Do not use spaces in file names.";
+		private const string Message = "File {0} has spaces in its path.";
```

Comment:
can you change this to "File '{0}' has spaces in its path."?

Reply:
OK, makes it more clear what the filename is, especially because it has a space in it ;-)